src/pages/TimeTableScheduler.tsx

```diff
@@ -397,6 +395,8 @@ const TimeTableSchedulerPage = () => {
     setCheckedCourses(newCheckedCourses)
   }
 
+  console.log(multipleOptions)
+
```

Comment:
Maybe you should also remove this console.log

Reply:
oopss